### ISSUE TITLE
Product stock availability column

### ISSUE THREAD

**Issue by ersinacar** [mage-eag/mage-enhanced-admin-grids#139]
Hi,
Great extension guys... I want to put product stock availability column to the product grid and I couldn't find it in the custom column list. Am I missed something or there is no such column ?

Thanks

**Comment by striperking** [comment]
Hi, 

Great extension, very usefull for my managers who just need a key 4-5 columns, however, we have a need to show qty and is_in_stock 

A lot of times qty is 10 but is_in_stock is "NO" and customers cannot order

I see there is some help above how to add this to the products_grid, but i cannot figure it out.  

For my inventory order, we just need sky, name, manufacture, qty, is_in_stock.  

So long story short, how do I add Stock Availability
